### PR TITLE
Added changes for Filereader object leak.

### PR DESCRIPTION
**Pull request by Vi-Nk**
Brief description of the PR: Added fix for possible Filereader resource leak during Exception in Initialization.

Created an independent FileReader object instead of a nested call from BufferedReader and closed in the finally block for GC cleanup.

Sonar Rule : “Resources should be closed” squid:S2095

Added formatting for Indentation and Javadoc

### CODE REVIEW

File: kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java [hunk 1]
```diff
@@ -36,12 +36,9 @@
 public class PppAuthSecrets {
 
     /*
-     * #Secrets for authentication using PAP/CHAP
-     * #client server secret IP addresses Provider
-     * ISP@CINGULARGPRS.COM * CINGULAR1 * #att
-     * mobileweb * password * #o2
-     * user * pass * #orange
-     * web * web * #vodaphone
+     * #Secrets for authentication using PAP/CHAP #client server secret IP addresses
```
Comment (MMaiero) [inline kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java:39]: Hi, thanks for the PR.
Are those changes to the comments needed?

Comment (Vi-Nk) [reply kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java:39]: Hi , without those changes , the build was throwing Tab-Space indentation problem for the whole file line by line. So , I had to apply Tab to Space conversion for the full file .

No change in content was done for the documentations.

**Comment by MMaiero** [comment]
Hi @Vi-Nk 
can you check the results of sonar? https://github.com/eclipse/kura/pull/3362/checks?check_run_id=2490555964
It is failing on your PR.

Thanks

**Comment by Vi-Nk** [comment]
@MMaiero Fixed. 
Somehow, Sonar was not recognizing the stream closure due to order of Buffered reader and Filereader. 
Changed order from -
```java
 finally {
            if (br != null) {
                try {
                    fr.close();
                    br.close();
                } catch (IOException ex) {
                }
            }
        }
```

to the below one has fixed it.
```java
 finally {
            if (br != null) {
                try {
                    br.close();
                    fr.close();
                } catch (IOException ex) {
                }
            }
        }
```

**Pull request merged**